FeaturedPhotoModal.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button, Image, StyleSheet, View, Text } from "react-native";
+import { StackParamList } from "./StackParamList";
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { StackNavigationProp } from "@react-navigation/stack";
+
+type FeaturedPhotoModalRouteProp = RouteProp<
+	StackParamList,
+	"FeaturedPhotoModal"
+>;
+
+type FeaturedPhotoModalNavigationProp = StackNavigationProp<
+	StackParamList,
+	"FeaturedPhotoModal"
+>;
+
+export const FeaturedPhotoModal = () => {
+	const { params } = useRoute<FeaturedPhotoModalRouteProp>();
+    const navigator = useNavigation<FeaturedPhotoModalNavigationProp>();
+	return (
+        <SafeAreaView style={styles.container}>
+            <View style={styles.buttonview}>
```

Comment:
You don't need this. Let React Navigation do the work for you!
1. Remove `headerShown:false` from `FeaturedPhotoModal` screen
2. Update this component. You won't need the navigation hook anymore.

```
  const { params } = useRoute<FeaturedPhotoModalRouteProp>();
  return (
    <SafeAreaView style={styles.container}>
      <Image
        source={{ uri: params.url }}
        alt={params.url}
        style={styles.image}
      />
    </SafeAreaView>
  );
```
3. Update your container style to `justifyContent: 'center'`
4. Remove spacingview and buttonview styles
5. ????
6. profit
![Simulator Screenshot - iPhone 14 Pro - 2023-08-19 at 11 42 42](https://github.com/spenc-neit/photogallery/assets/46625943/560ff639-cd89-402b-b2ae-d81ba7c663e4)